src/odemis/gui/cont/settings.py

```diff
@@ -719,3 +723,203 @@ def __init__(self, tab_panel, tab_data):
                     continue
                 conf = self._va_config[comp.role].get(a)
                 self._spect_setting_cont.add_axis(a, comp, conf)
+
+
+# constants defining the displayed status in the GUI
+SAVE = "Save"
+LOAD = "Load"
+ERROR = "Error"
+STATUS = "Status"
+
+
+class StreakCamAlignSettingsController(SettingsBarController):
+    """
+    """
+    def __init__(self, tab_panel, tab_data):
+        super(StreakCamAlignSettingsController, self).__init__(tab_data)
+        self.panel = tab_panel
+        main_data = tab_data.main
+        self.streak_ccd = main_data.streak_ccd
+        self.streak_delay = main_data.streak_delay
+        self.streak_unit = main_data.streak_unit
+        self.streak_lens = main_data.streak_lens
+
+        self._calib_path = None  # path to the trigger delay calibration folder
```

Comment:
I think that if you initialise it directly with `get_picture_folder()` here, you can drop the two `if not self._calib_path`.